docs/tc53.md

```diff
@@ -2448,9 +2451,64 @@ None
 | brightness | no | 0.0 to 1.0 | |
 | flip | no | "", "h", "v", or "hv" | |
 
+<a id="alg-sensor-prov-class-pattern"></a>
 ## Provenance Sensor Class Pattern
 
-<!-- to do -->
+### `configure`(*options*)
+1. Execute all steps of the Peripheral Class Pattern configure method
```

Comment:
This should execute all steps of the Sensor Class Pattern (which it inherits from) rather than the Peripheral Class Pattern (which the Sensor Class Pattern inherits from). While it makes no functional difference now, it could in the future if Sensor Class Pattern configure adds additional steps.